content/quickstart.md

```diff
@@ -345,3 +345,97 @@ and sends invoices to clients on the same schedule.
 
 [harvest]: https://www.getharvest.com/
 [accounts]: mailto:accounts@oddbird.net
+
+## Success Criteria:
+
+This guide will help you understand the expectations for your role at OddBird.
+The criteria are divided into three groups: Work Quality, Communication, and
+Learning and Growth. Each group has three levels of performance: Under performs,
+Meets, and Exceeds. You can use this guide to assess your own performance,
+identify your strengths and areas for improvement, and set your goals and action
+plans.
+
+### Work Quality
+
+#### Under performs
+
+- Lacks initiative and waits for directions or assignments on high priority work
+- Produces low-quality or incomplete work that requires excessive rework or
+  refactoring from others
+- Uses outdated or inappropriate tools, technologies, or methodologies
+- Avoids or gives up on challenges
+- Does not plan or dissect big assignments into smaller tasks
```

Comment:
```suggestion
- Does not plan ahead or break big assignments into smaller tasks
```

Reply:
I think "plan ahead" has a slightly different meaning than the intended one, and I personally prefer "dissect" rather than "break" here. 🤷 Maybe there's a synonym we can find that would be clearer than "plan"?

Reply:
Would "Does not create plans for big assignments, or dissect them into smaller tasks" work?

Reply:
I really like James' wording suggestion here. This can be applied to all the 3 sections where it comes up.

Reply:
Fixed in 9c8418ba1f92581a5310d6aa36bddf39c5f7d104